pepsico/projections/maproom.py

```diff
@@ -347,19 +348,19 @@ def seasonal_change(
         data = data - ref
         if variable in ["hurs", "huss", "pr"]:
             data = 100. * data / ref
-            data["units"] = "%"
+            data.attrs["units"] = "%"
         return data.rename({"X": "lon", "Y": "lat"})
 
 
-    def map_attributes(variable, data=None):
+    def map_attributes(data, to_dash_leaflet=False):
```

Comment:
I don't love the `to_dash_leaflet` option. I think it's better to have this function always return the same representation, and have the client translate it to dash-leaflet representation if that's what's needed.

The general principle is to use a uniform representation throughout our application, and when interfacing with other applications or libraries that use a different representation, do the translation at the interface, rather than letting the complexity of multiple representations propagate through our code.

Reply:
taken out